ext/standard/basic_functions.c

```diff
@@ -141,6 +141,7 @@ static void user_shutdown_function_dtor(zval *zv);
 static void user_tick_function_dtor(user_tick_function_entry *tick_function_entry);
 
 static const zend_module_dep standard_deps[] = { /* {{{ */
+	ZEND_MOD_REQUIRED("uri")
```

Comment:
If `uri` is required, it should not require the `--enable-uri` flag to build it. It should always be built, like `ext/random`.

Reply:
Ah yes, good catch!